### DISCUSSION TITLE
Populate records using a computed column without raw query

### DISCUSSION THREAD

**Discussion by joseph-hailu-dd** [sequelize/sequelize#14338] [Help & Questions] [answered]
Is there a way to update a column using a computed row like row_number() without using raw sql? 
Currently my solution is
```
sequelize.query(
   "UPDATE users 
   SET new_unique_not_null_col = 'val' || row_value
   FROM
     (SELECT u.id, ROW_NUMBER() OVER (ORDER BY id) AS row_value
      FROM users u) usr
   WHERE usr.id = users.id"
)
```
could i somehow use sequelize.changeColumn() for this instead?

**Comment by ephys** [comment, marked as answer]
Unfortunately, `UPDATE ... FROM` is not supported at all in Sequelize currently. You only solution is to use a raw query.

I'm pretty sure I've seen an feature request for it, but I can't find it
